test/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+     <body>
+          <head>
+               <link rel="stylesheet" href="stylesheet.css"/>
+          </head>
+          <h1 style="text-align: center;">Roopali Sharma</h1>
+          <div class="background" style="text-align: right;">
+               <a class="a avtive" href="a">Home</a>
+               <a class="b" href="#">About Me</a>
+               <a class="c" href="#">Git Hub</a>
+               <a class="d" href="#">Linked In</a>
```

Comment:
these links all go to the home page. you want them to go to the corresponding section. 
```
<a class="a active" href="#">Home</a> // home link
<a class="b" href="#AboutMe">About Me</a> // link to about me section on your page
<a class="c" href="https://github.com/roopali1022/" target='_blank'>Git Hub</a> // your github page
```